docs/adr/0031-process-evolution.md

```diff
@@ -0,0 +1,199 @@
+# FxA Process Evolution
+
+- Status: proposed
+- Deciders: Wil Clouser, Ivo Plamenac, Jon Buckley, Dustin Lactin
+- Date: 2022-06-02
+
+## Context and Problem Statement
+
+We have a [documented process](https://mozilla.github.io/ecosystem-platform/reference/team-processes/development-process)
+today (see also the [release owner duties](https://mozilla.github.io/ecosystem-platform/reference/team-processes/release-process)),
+but it’s complex and can be error-prone. The release process was built before we have the tools
```

Comment:
Can we elaborate on what it is about our current process that makes it complex and error-prone? I think we all have the sense that this is true, but it's not clear to me how any of these options are less complex or less error prone without a direct comparison (e.g. some options avoid the step of merging a release back into main, which takes time and has these risks, ...).